src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -1334,6 +1335,43 @@ public void checkOutFailsWhenEventPublishingFailsWithBadRequestError() {
     assertThat(response.getBody(), containsString(
       "Error during publishing Event Message in PubSub. Status code: 400"));
   }
+
```

Comment:
Sorry for the confusion, I've changed the description in the story. This PR's purpose is to accumulate validation errors to be able to return multiple errors instead of failing with the first one. This is the base for the next check-out override story (CIRC-1061) that will deal with the overrides themselves, so overrides and overridable errors are actually out of scope here and I've removed all mentions of them from this PR.